content/en/real_user_monitoring/guide/proxy-rum-data.md

```diff
@@ -57,33 +89,60 @@ window.DD_RUM &&
+- `parameters`: the parameters of the datadog requests, ex: `ddsource=browser&...`
 
-The site parameter is an SDK [initialization parameter][1]. Datadog intake origins for each site are listed below:
+{{< tabs >}}
+{{% tab "NPM" %}}
 
-| Site    | Site Parameter            | Datadog intake origin                      |
-|---------|---------------------------|--------------------------------------------|
-| US1     | `datadoghq.com` (default) | `https://browser-intake-datadoghq.com`     |
-| US3     | `us3.datadoghq.com`       | `https://browser-intake-us3-datadoghq.com` |
-| US5     | `us5.datadoghq.com`       | `https://browser-intake-us5-datadoghq.com` |
-| EU1     | `datadoghq.eu`            | `https://browser-intake-datadoghq.eu`      |
-| US1-FED | `ddog-gov.com`            | `https://browser-intake-ddog-gov.com`      |
-| AP1     | `ap1.datadoghq.com`       | `https://browser-intake-ap1-datadoghq.com` |
+```javascript
+import { Datacenter, datadogRum } from '@datadog/browser-rum';
+
+datadogRum.init({
+    applicationId: '<DATADOG_APPLICATION_ID>',
+    clientToken: '<DATADOG_CLIENT_TOKEN>',
+    site: '<DATADOG_SITE>',
+    proxy: (options) => `https://www.proxy.com/foo${options.path}/bar?${options.parameters}`,
+});
+```
+
+{{% /tab %}}
+{{% tab "CDN async" %}}
+```javascript
+window.DD_RUM.onReady(function() {
+    window.DD_RUM.init({
+        clientToken: '<CLIENT_TOKEN>',
+        applicationId: '<APPLICATION_ID>',
+        proxy: (options) => `https://www.proxy.com/foo${options.path}/bar?${options.parameters}`,
+    })
+})
+```
+{{% /tab %}}
+{{% tab "CDN sync" %}}
+
+```javascript
+window.DD_RUM &&
+    window.DD_RUM.init({
+        clientToken: '<CLIENT_TOKEN>',
+        applicationId: '<APPLICATION_ID>',
+        proxy: (options) => `https://www.proxy.com/foo${options.path}/bar?${options.parameters}`
+    });
+```
+
+{{% /tab %}}
+{{< /tabs >}}
 
-The Datadog intake origin corresponding to your site parameter should be defined in your proxy implementation.
+For example, with a `site` set to `datadoghq.eu` and the `proxy` configuration from the example, the RUM Browser SDK sends requests to an URL that looks this: `https://www.proxy.com/foo/api/v2/rum/bar?ddsource=browser`. The proxy will need to forward the request to the URL `https://browser-intake-datadoghq.eu/api/v2/rum?ddsource=browser`.
 
-For example, with a `site` set to `datadoghq.eu` and a `proxy` set to `https://example.org/datadog-intake-proxy`, the RUM Browser SDK sends requests to an URL that looks this: `https://example.org/datadog-intake-proxy?ddforward=%2Ffoo%2Fv2%2Fbar%3Fbaz%3D1`. The proxy will need to forward the request to the URL `https://browser-intake-datadoghq.eu/foo/v2/bar?baz=1`.
+**Note:** Some privacy blockers already target the intake [url patterns][2], so you may want to take that into account when building your proxy URL.
+**Note:** The `proxy` function is called for each request, so it should avoid any heavy computation.
```

Comment:
I don't know why but the notes are not displayed as expected:
![image](https://github.com/DataDog/documentation/assets/4342515/8851ae98-4bd0-4293-9fff-d40c6f997d95)

Reply:
Markdown expect a blank line between paragraphs